core/domain/src/main/java/com/itenum/domain/Email.kt

```diff
@@ -0,0 +1,3 @@
+package com.itenum.domain
+
+data class Email (var to: String, var subject: String, var body: String)
```

Comment:
You only need to create one model, no need to create 2 models
You create only one contact model and in this model you add all the information